notebooks/on_flight/README.md

```diff
@@ -0,0 +1,92 @@
+# Flight updates package for ORCESTRA
+
+### Purpose
+
+This small package plots various satelite and forecast products that might be useful for the flight crew during a flight. It will plot the most recent data available, so can be updated as the flight proceeds.
+
+### Acknowledgements
+
+The script makes liberal use of the "goes.py" set of functions that I shamelessly stole from the weather_briefing_package written by Lukas, Tobi, Henning, and Divya (maybe others). A modified version of these functions are included with this package.
+
+
+### Usage
+
+0) Copy the file HALO-flightID.ipynb to another file, replacing "flightID" with the actual flight ID.
+
+1) Set the flight takeoff time and forecast initialisation time in the second code block of the iphython notebook.
+
+2) Download the flight plan from https://orcestra-campaign.org/operation/halo.html. Copy the first code cell (between the definition of "radius" to the defintion of "path" 
+
+3) Running the iPython notebook will download various data and save figures to directories detailed below
+
+4) To send a figure to the flight crew, open it in preview (Mac) and export it to jpeg with the quality set such that the size is smaller than 200 kB.
```

Comment:
This step seems very inconvenient... I think it is much easier (and likely better quality) if you create the images with an adequate size directly. Just ping me, if you need help with that ☺️

Reply:
I guess the idea is that the ground support will be making many more figures and looking at them in much more detail than the flight crew, so they want them at a decent resolution. Then they choose one or two to send, so it is not so bad to resize them. But I am completely open to better ways.